### ISSUE TITLE
Improve files:scan for groups containing a comma

### ISSUE THREAD

**Issue by mmattel** [owncloud/core#32215]
10.0.9
It would be good to harmonize the ability to enable groups containing a comma in their name.

Based on a comment of @phil-davis in https://github.com/owncloud/password_policy/pull/77#pullrequestreview-142276087

**snip...**
In core repo ``core/Command/User/Add.php`` the option is called just group because you specify just 1 group name after each --group :

--group Finance,Sydney --group Finance,Berlin --group MyFriends
**snip...**

Maybe true for other occ cammands where this change would be benificial

In case of a change, doc relevant for all commands affected

@PVince81

**Comment by phil-davis** [comment]
I thought that I had recently found and fixed the places that did not cope with group names containing a comma, but not yet! As you point out, and I noticed today, ``files:scan`` does not cope with specifying a group name that contains a comma.

Maybe there are other places? 

I believe the webUI is OK - user management (adding a group, putting users in a group, deleting a group...), sharing with a group, blacklist groups from sharing.
